### ISSUE TITLE
FYI: macOS ld: warning: -undefined dynamic_lookup may not work with chained fixups

### ISSUE THREAD

**Issue by rwgk** [cctbx/cctbx_project#815]
To hopefully save you some time digging this up:

* https://github.com/cctbx/cctbx_project/blob/b6b8e3bb27e2a75a076155baa921e478fcb71c0d/libtbx/SConscript#L799

That's deprecated in Xcode 14.

Solution: https://github.com/python/cpython/issues/97524

**Comment by bkpoon** [comment]
Thanks for the heads up. Does the warning only show up when `-mmacosx-version-min=12.0`? I'm not seeing the warning in our builds, but we set the minimum to 10.9 for Intel and 11.0 for Apple Silicon.

**Comment by rwgk** [comment]
> Thanks for the heads up. Does the warning only show up when `-mmacosx-version-min=12.0`? I'm not seeing the warning in our Xcode 14 builds on Monterey, but we set the minimum to 10.9 for Intel and 11.0 for Apple Silicon.

Sorry I don't know. There is some more background here: https://github.com/pybind/pybind11/pull/4301

My recommendation: just wait until you actually see the warning, then use the links here to see what others did.

**Comment by bkpoon** [comment]
Yeah, that's what I was thinking too. I'll update to Ventura when the .1 release is available and see if anything changes as newer versions of Xcode come out.

I also pinged the `conda-forge` folks to see if they know anything more.
